libkbfs/folder_branch_ops.go

```diff
@@ -2204,6 +2218,20 @@ func (fbo *folderBranchOps) createEntryLocked(
 		newBlock = &FileBlock{}
 	}
 
+	// Passthrough journal writes temporarily.
+	if excl == WithExcl {
+		if jServer, err := GetJournalServer(fbo.config); err == nil {
+			if err := jServer.DisableWrites(ctx, fbo.id()); err != nil {
```

Comment:
Oh, hrm, that's a very good point.  I guess I was thinking of a slightly different architecture when I started, but it turned into something else.  You're right, a complete disable should work, I'll do that instead.